src/blocks/Media/Media.tsx

```diff
@@ -4,19 +4,29 @@ import Media from '../../components/Media/Media';
 import MediaBase from '../../components/MediaBase/MediaBase';
 import {useTheme} from '../../context/theme';
 import {MediaBlockProps} from '../../models';
-import {getThemedValue} from '../../utils';
+import {block, getThemedValue} from '../../utils';
+
+import './Media.scss';
+
+const b = block('media-block');
 
 export const MediaBlock = (props: MediaBlockProps) => {
-    const {media} = props;
+    const {media, border = 'shadow', disableShadow} = props;
+
+    const mediaBaseShadow = disableShadow || border !== 'shadow';
```

Comment:
We need another naming for `mediaBaseShadow` because it is boolean flag

Reply:
changed